src/review/commands/share/hookHandlers/pushHookHandler.ts

```diff
@@ -29,10 +29,8 @@ export async function pushHookHandler(
     return;
   }
 
-  const branchName = ref.split('/').pop() as string;
```

Comment:
Looking at the test logs, if the branch is `foo/bar`, then `ref` is going to be `refs/branch/foo/bar`. This isn't right.